components/ItemList.js

```diff
@@ -1,25 +1,36 @@
 import { Fragment } from 'react';
 import ItemCard from './ItemCard';
 import styled from 'styled-components';
+import Link from 'next/link';
 
-export default function ItemList({ items, onRemoveItem, onEditItem }) {
+export default function ItemList({
+  currentItems,
+  onRemoveItem,
+  onEditItem,
+  storages,
+}) {
   return (
-    <StyledList>
-      {items.map((item) => {
-        return (
-          <Fragment key={item.id}>
-            <ItemCard
-              id={item.id}
-              name={item.name}
-              date={item.date}
-              quantity={item.quantity}
-              onRemoveItem={onRemoveItem}
-              onEditItem={onEditItem}
-            />
-          </Fragment>
-        );
-      })}
-    </StyledList>
+    <>
+      <Link href="/">Go back</Link>
+      <StyledList>
+        {currentItems.map((item) => {
+          return (
+            <Fragment key={item.id}>
+              <ItemCard
+                id={item.id}
+                name={item.name}
+                date={item.date}
+                quantity={item.quantity}
+                storage={item.storage}
+                onRemoveItem={onRemoveItem}
+                onEditItem={onEditItem}
+                storages={storages}
+              />
+            </Fragment>
+          );
```

Comment:
You don't need a `<Fragment>` here - just add the `key` to `ItemCard` and return it directly.